searchMatrix.py

```diff
@@ -0,0 +1,25 @@
+# Time Complexity : O(m*n)
+# Space Complexity : O(1)
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No..but will have to change this for optimal
+#  solution with two pointers
+
```

Comment:
use the approach thought in class and update the solution.

Reply:
Will do!